### ISSUE TITLE
Cannot fix 'code blocks should be indented'

### ISSUE THREAD

**Issue by scpketer** [remarkjs/remark-lint#189]
remark shows a warning on a code block, and I've got no clue **how** it should be indented not to cause the warning.

Sample:
\```python
from lib import Class1, Class2
c1 = Class1()
c2 = Class2()```

**Comment by Murderlon** [comment]
hi @scpketer, perhaps this helps: https://github.com/remarkjs/remark-lint/tree/master/packages/remark-lint-code-block-style

**Comment by scpketer** [comment]
@Murderlon Nope, it didn't

**Comment by Murderlon** [comment]
One of these two should resolve your error.

````markdown
```
from lib import Class1, Class2
c1 = Class1()
c2 = Class2()
```
````

```markdown
    from lib import Class1, Class2
    c1 = Class1()
    c2 = Class2()
```

**Comment by scpketer** [comment]
Problem is that I need to make a code block for the github's markdown, which will not recognize the indented lang-less code block

**Comment by Murderlon** [comment]
Create a `.remarkrc` file with you preferred settings. I believe the linter should pick it up.

https://github.com/remarkjs/remark-lint#configuring-remark-lint

**Comment by scpketer** [comment]
Okay, I've resolved that. Just had to edit a preset. Thanks anyways
